topics/proteomics/tutorials/DIA_lib_OSW/tutorial.md

```diff
@@ -159,14 +159,16 @@ The PTXQC software ({% cite Bielow2015 %}) was built to enable direct proteomcs
 >        - {% icon param-file %} *"Infile"*: `MaxQuant_Evidence`
 >    - *"With following condition"*: `len(c9.split(';')) < 2`
 >    - *"Number of header lines to skip"*: `1`
+>    - Rename to 'Filter on MaxQuant_Evidence'
 >
 > 2. {% tool [Filter](Filter1) %} with the following parameters:
 >        - {% icon param-file %} *"Infile"*: `MaxQuant_MSMS`
 >    - *"With following condition"*: `len(c12.split(';')) < 2`
 >    - *"Number of header lines to skip"*: `1`
+>    - Rename to 'Filter on MaxQuant_MSMS'
```

Comment:
Please make this a separate bullet point, since now it looks as if it is a parameter of the Filter tool.